chat_shell/chat_shell/messages/converter.py

```
@@ -103,50 +94,83 @@ def build_messages(
+                            image_bytes = base64.b64decode(base64_data)
+                            if len(image_bytes) > MAX_IMAGE_SIZE_BYTES:
+                                compressed_bytes = MessageConverter._compress_image(
+                                    image_bytes, mime_type
+                                )
+                                compressed_base64 = base64.b64encode(
+                                    compressed_bytes
+                                ).decode("utf-8")
+                                image_url = (
+                                    f"data:{mime_type};base64,{compressed_base64}"
+                                )
+                    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)